avm/res/network/virtual-network/main.json

```diff
@@ -5,8 +5,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.29.47.4906",
-      "templateHash": "17645744687566805489"
+      "version": "0.21.1.54444",
```

Comment:
Hey @elbatane please don't downgrade the bicep version. 
You can avoid to recompile the json as the update only affects the readme file

Reply:
Done!